File: app/src/main/java/com/avoid/playtolearn/models/Category.java
package com.avoid.playtolearn.models;

import java.io.Serializable;

public enum Category implements Serializable{
    Loops(1, "Loops"),
    Conditional_Structures(2, "Conditional structures");

    private int id;
    private String name;

    Category(int id, String name){
        this.id = id;
        this.name = name;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/com/ivantha/playtolearn/model/Category.kt
package com.ivantha.playtolearn.model

class Category {
    var id: String? = null
    var name: String? = null

    constructor()

    constructor(id: String) {
        this.id = id
    }

    constructor(id: String, name: String) {
        this.id = id
        this.name = name
    }

}
